src/components/menus/directory-tree-contextual-menu.js

```diff
@@ -234,6 +234,20 @@ const DirectoryTreeContextualMenu = (props) => {
                                     });
 
                                 break;
+                            case ElementType.SECURITY_ANALYSIS_PARAMETERS:
```

Comment:
also here the calls to duplicate params can be reused for security_analysis and voltage_init